method_name: "frame_infer"

dataset_frame_path: "/home/ssd_dir/dataset_image_to_gravity_ozaki/stick/1cam/dkan_outdoor_cw"
csv_name: "imu_camera.csv"

weights_top_path: "/home/bnn_attitude_predictor_with_image/test2/weights/"
weights_file_name: "regression1775train318valid224resize0.5mean0.5stdAdam1e-05lrcnn0.0001lrfc70batch50epoch.pth"

log_file_path: "/home/bnn_attitude_predictor_with_image/test2/infer_log/frame_infer"
log_file_name: "frame_infer_result.csv"